internal/common/eventutil/eventutil.go

```diff
@@ -1043,6 +1043,45 @@ func EventSequenceFromApiEvent(msg *api.EventMessage) (sequence *armadaevents.Ev
 		// Do nothing; there's no corresponding Pulsar message.
 	case *api.EventMessage_Updated:
 		// Do nothing; we're not allowing arbitrary job updates.
+	// TODO: add case for Preempted event
+	case *api.EventMessage_Preempted:
+		sequence.Queue = m.Preempted.Queue
+		sequence.JobSetName = m.Preempted.JobSetId
+
+		preemptedJobId, err := armadaevents.ProtoUuidFromUlidString(m.Preempted.JobId)
+		if err != nil {
+			return nil, err
+		}
+
+		event := &armadaevents.EventSequence_Event_JobPreempted{
+			JobPreempted: &armadaevents.JobPreempted{
+				PreemptedJobId:         preemptedJobId,
```

Comment:
Do we need `PreemptedJobSetId` and `PreemptedJobQueue` as these are presumably already available from the eventSequence

Reply:
well no, I will remove them